content/blog/2025/04/kubewarden-1.24-release.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Kubewarden 1.24 release
+authors:
+  - Víctor Cuadrado Juan
+date: 2025-04-30
+---
+
+The wait is over—Kubewarden 1.24 has arrived! We have some easter eggs for you
+in this one.
+
+## Promoting our policies to v1.0.0
+
+In the past, we consciously picked semver `0.X.Y` for policy versions as that
+meant that the policy API for the user (in this case, the policy
+`spec.settings`) was not considered stable. To be clear, we always ensure
+the policy API and ABI between policy-server and the policy is always
+compatible.
+
+We haven't needed to review this over the past few years,
+so policies kept getting 0.X.0 minor releases, with none breaking backward
+compatibility.
+
+So, it is about time we proudly declare that our policies are semver
+`v1.0.0`!
+
+We have released most of our policies, those we consider stable,
+as `v1.0.0`. This shows our commitment over this time,
+now paraded in Semantic Versioning. You can find
+[here](https://github.com/kubewarden/community?tab=readme-ov-file#repositories)
+a list of policy repositories and their maturity level; note that the
+majority are declared stable. Others, like our `kyverno-dsl-policy`, being
+experimental, are released with version `0.1.0`.
+
+Benefits to using `v1.0.0` are:
+
+- Policy OCI artifacts are labeled with `org.opencontainers.image.*`. This
+  informs tools such as the Renovate bot, where the repo and changelog of the policy
+  live, which provides useful metadata when letting bots bump the policies you consume.
```

Comment:
```suggestion
- Policy OCI artifacts are labeled with `org.opencontainers.image.*` labels. This
  information can be used by automation tools like Renovate or Dependabot to react
  to new policy releases. For example, it would be possible to have Renovate crate
  pull requests to bump the policy version consumed by a `ClusterAdmissionPolicy`
  definition.
```